maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .book-info-container {
+    flex-grow: 0;
+    margin-right: 0;
+  }
+
+  .book-title-container {
+    margin-bottom: 0;
+  }
+
+  .book-title {
+    font-size: 20px;
+    margin-right: 10px;
+  }
+
+  .book-gender {
+    font-size: 16px;
+  }
+
+  .book-detail-name {
+    font-size: 14px;
+    margin-right: 10px;
+    margin-bottom: 20px;
+  }
+
+  .book-detail-value {
+    font-size: 14px;
+    margin-bottom: 0;
+  }
+
+  .book-container {
+    align-items: center;
+    box-shadow: 4px 4px 10px 0 $black-05;
+    flex-direction: column;
+    max-width: 330px;
+    margin: 30px auto;
+    padding: 30px;
+    position: relative;
+    width: 100%;
+    &::after {
+      content: '';
+      background: url('./assets/book-badge.png') no-repeat center/contain;
+      height: 70px;
+      left: 220px;
+      position: absolute;
+      top: 20px;
+      transform: rotate(15deg);
```

Comment:
The following properties are already set outside the media query: 

``` 
content: '';
background: url('./assets/book-badge.png') no-repeat center/contain;
left: 220px;
position: absolute;
transform: rotate(15deg);
```

Don't re write them unless you're going to override them for other value